SConstruct

```diff
@@ -340,6 +340,16 @@ def scons(): # pylint: disable=too-many-locals
 
     env = Environment(TOOLS=['extra', 'default', 'textfile'])
 
+    # Scons strips out the environment, however to be able to build daos using the interception
+    # library we need to add a few things back in.
+    if 'LD_PRELOAD' in os.environ:
+        env['ENV']['LD_PRELOAD'] = os.environ['LD_PRELOAD']
+
+        for key in ['D_LOG_FILE', 'DAOS_AGENT_DRPC_DIR', 'D_LOG_MASK', 'DD_MASK', 'DD_SUBSYS']:
+            value = os.environ.get(key, None)
+            if value is not None:
+                env['ENV'][key] = value
```

Comment:
(pylint-invalid-sequence-index) Sequence index is not an int, slice, or instance with __index__